domain/memos/repository.go

```diff
@@ -0,0 +1,21 @@
+package memos
+
+import (
+	"time"
+)
+
+type Memo struct {
```

Comment:
What do you think of moving this part to `memos/entity.go`?

Reply:
i agree

Reply:
Great, I moved Memo struct to `entity.go` file :)